addon/components/materialize-copyright.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+import layout from '../templates/components/materialize-copyright';
+
+export default Ember.Component.extend({
+  init: function() {
+    this._super();
+    if (this.get('startYear') != null) {
+      Ember.assert('Property startYear must be less than or equal to the current year.',
+        this.get('startYear') <= new Date().getFullYear());
+    }
+  },
+  layout: layout,
+  classNames: ['footer-copyright'],
+  startYear: null,
+  text: null,
+  date: function () {
+    var returnDate;
+    if (this.get('startYear') === null || this.get('startYear') === new Date().getFullYear()) {
```

Comment:
you could call `this.get('startYear')` once, and use the same local variable throughout this function body